### ISSUE TITLE
Enable project tree view

### ISSUE THREAD

**Issue by EricJizbaMSFT** [microsoft/vscode-azurefunctions#1163]
I added preview support [here](https://github.com/Microsoft/vscode-azurefunctions/pull/1162), but there's some known issues that need to be addressed before we can remove the feature flag. Using this to track all those issues

Minimum features required before enabling:
- [x] detect new projects: https://github.com/Microsoft/vscode-azurefunctions/issues/1164
- [ ] support C# (or at least show a warning) https://github.com/Microsoft/vscode-azurefunctions/issues/1165
- [ ] copy a function's http url https://github.com/Microsoft/vscode-azurefunctions/issues/1167
- [x] tree item picker https://github.com/Microsoft/vscode-azurefunctions/issues/1170

Nice to have, but not required to enable:
- [ ] show local app settings https://github.com/Microsoft/vscode-azurefunctions/issues/1166
- [ ] show local proxies https://github.com/Microsoft/vscode-azurefunctions/issues/1168
- [ ] maybe be in file explorer https://github.com/Microsoft/vscode-azurefunctions/issues/1169
